docs/en/build_and_run.md

```diff
@@ -0,0 +1,162 @@
+
+# Build compilation environment
+
+**Note:**
+1. If you just want to experience the deployment and basic functions of CURVE, **you do not need to compile CURVE**, please refer to [Single-machine deployment](deploy.md).
+2. This document is only used to help you build the curve code compilation environment, which is convenient for you to participate in the development, debugging and packaging test of curve.
```

Comment:
"curve" should be "Curve" or "CURVE" be consistent

Reply:
fix